### ISSUE TITLE
Flutter Text

### ISSUE THREAD

**Issue by NeoTmg** [Shahxad-Akram/flutter_tex#66]
when i use attribute android:repalce="android:label"  in android manifest file to open the external activity in my project than flutter_tex will not rendering the equestions. if i remove android:replace="android:label" then it will render. otherwise it will cause initview not initialize....

**Comment by shah-xad** [comment]
This detail isn't enough please show some code to reproduce it.

**Comment by NeoTmg** [comment]
This is my android manifest file where i use android:replace="android:label" and when i use this or merger in manifest then equation will not going to render. 
<application
        
        android:usesCleartextTraffic="true"
        android:name="io.flutter.app.FlutterApplication"
        android:label="@string/app_name"
        **tools:replace="android:label"**
        android:supportsRtl="true"
        android:icon="@mipmap/ic_launcher"
        android:hardwareAccelerated="true"
        >

My Code :  TeXView(
                        
                        renderingEngine: TeXViewRenderingEngine.katex(),
                        child: TeXViewDocument(new_new_new_question,style: TeXViewStyle.fromCSS(
                                            'padding: 15px;textAlign: TeXViewTextAlign.justify'),
                        ),
                        style: TeXViewStyle(
                          //textAlign: TextAlign.justify
                        ),)

Output : 
W/ContentCatcher(29944): Failed to notify a WebView
W/Looper  (29944): Slow Looper main: doFrame is 550ms late because of 13 msg
**I/chromium(29944): [INFO:CONSOLE(1)] "Uncaught ReferenceError: initView is not defined", source: chrome-error://chromewebdata/ (1)**
I/chatty  (29944): uid=10502(com.qualityitsolution.eduspell) identical 3 lines
I/chromium(29944): [INFO:CONSOLE(1)] "Uncaught ReferenceError: initView is not defined", source: chrome-error://chromewebdata/ (1)

![output](https://user-images.githubusercontent.com/37269529/85649962-1e837b80-b6c4-11ea-80f6-0d7133756b99.jpg)

**Comment by DarkKnight1005** [comment]
By the way, however i did everything as in readme i have almost the same issue as this one. I see no error in debug console but tex does not render and i have these green lines instead of equations. Furthermore, when i write in console .\gradlew build it throws an error because of 'webview' and my build fails . I really do not understand why i can execute my app while building via '.\gradlew build ' i have fail.

Here is where build fails
`> Task :webview_flutter:lint FAILED
Ran lint on variant profile: 9 issues found
Ran lint on variant debug: 9 issues found
Ran lint on variant release: 9 issues found
Wrote HTML report to file:///D:/ASprojects/math_upgrader_app/build/webview_flutter/reports/lint-results.html
Wrote XML report to file:///D:/ASprojects/math_upgrader_app/build/webview_flutter/reports/lint-results.xml

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':webview_flutter:lint'.
> Lint found errors in the project; aborting build.

  Fix the issues identified by lint, or add the following to your build script to proceed with errors:
  ...
  android {
      lintOptions {
          abortOnError false
      }
  }
  ...

  Errors found:

  D:\flutter\flutter\.pub-cache\hosted\pub.dartlang.org\webview_flutter-0.3.22+1\android\src\main\java\io\flutter\plugins\webviewflutter\FlutterWebView.java:313: Error: Call requires API level 19 (current min is 16): android.webkit.WebView#setWebContentsDebuggingEnabled [NewApi]
            webView.setWebContentsDebuggingEnabled(debuggingEnabled);
                    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  D:\flutter\flutter\.pub-cache\hosted\pub.dartlang.org\webview_flutter-0.3.22+1\android\src\main\java\io\flutter\plugins\webviewflutter\FlutterWebView.java:343: Error: Call requires API level 17 (current min is 16): android.webkit.WebSettings#setMediaPlaybackRequiresUserGesture [NewApi]
      webView.getSettings().setMediaPlaybackRequiresUserGesture(requireUserGesture);
                            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~



* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 7m 24s`

**Comment by shah-xad** [comment]
@DarkKnight1005 According to your Error Report your min API level is 16, you should have at least 19 or above.

**Comment by DarkKnight1005** [comment]
> @DarkKnight1005 According to your Error Report your min API level is 16, you should have at least 19 or above.

No, my min API level is 21, but i figured out why build fails.Inside webview plugin there is own gradle file, in this file min api level was setted up to 16. I just change it to 21 and build successful, but the point that i still have a problem, equations do not render.

**Comment by Shahxad-Akram** [comment]
Sorry, it's been so long, but I hope it's been fixed. Please don't hesitate to open a new issue if the issue continues.
